pike/model.py

```diff
@@ -53,6 +53,7 @@
 import time
 import operator
 import contextlib
+import polling2
```

Comment:
i think you need to add `polling2` to the `install_requires` in `setup.cfg`  if you're going to import it here.